src/app/components/normal-component/pick-game/pick-game.component.ts

```diff
@@ -174,6 +177,8 @@ class GameDescription {
 
     public static readonly GO: Localized = () => $localize`The oldest strategy game still practiced widely. A territory control game.`;
 
+    public static readonly HEXAGONAL_CONNECTION: Localized = () => $localize`An hexagonal alignment game with weird "diagonals"!`;
```

Comment:
ChatGPT

The correct phrase is "a hexagonal grid."

In English, the use of "a" or "an" depends on the sound that follows. "A" is used before words that begin with a consonant sound, and "an" is used before words that begin with a vowel sound. Although "hexagonal" starts with the letter "h," which is a consonant, the "h" is pronounced in this case, making the initial sound a consonant sound. Therefore, you should use "a" before "hexagonal grid."

Reply:
"a h" ok, mais "un jeu d'alignement hexagonal" et "un quadrillage hexagonal" c'pas la même phrase :D grid et alignment ça a rien à voir si ?